uPy/main.py

```diff
@@ -71,6 +71,11 @@ def sd_gdt_func(timer):
 wifi = "/sdcard/wifi.log"
 archive = "/sdcard/data.log"
 unsent = "/sdcard/buffer.log"
+blacklist = "/sdcard/blacklist.log"
+current_ap = "/sdcard/SSID.log"
+
+with open(blacklist, "a+"):
```

Comment:
why is this done this way over:  filepointer = open(blacklist, "a+")

Reply:
Add comments Mr. Compton

Reply:
Partial credit!!!

Reply:
add a comment explaining this